drafts/build_a_dao_on_fuel/draft.md

```diff
@@ -0,0 +1,1095 @@
+[note]: <> ("note to self and file editor, there are certain comments left throughout this file, for the sake of understanding the author's reason for using a certain method of approach, pls feel free to delete once acknowledged or reply edit with a better approach 🥂")
+
+# How to build a DAO on the Fuel Blockchain.
+
+## Introduction
+
+A Decentralized Autonomous Organization (DAO) is considered quite an essential infrastructure in any blockchain.
+
+It is a Decentralized technology that blockchains incorporate into their validation system to automate governance rules.
+Precisely, it is an automated organization represented by rules encoded in a computer program that is transparent, controlled by the organization members, and not influenced by any central authority.
+
+On completing this article you will learn how to build a full-stack DAO on the fuel blockchain, you will touch down on the following concepts of development;
+
+1. Writing a DAO smart contract using the sway language.
+2. Wallet configuration
+3. Compiling and deploying to the fuel blockchain (testnet).
+4. Creating your DAO front end including contract calls from your deplooyed contract. 
+
+## Prerequisites
+
+To follow through this tutorial seamlessly, it is expected that you have the following checked out.
+
+1. Installation of dependencies like `fuel`, `sway` extension on vscode, `rustc`, `cargo`, `rustup` `cargo-generate`.
+2. Node version ^18.18.2 || ^20.0.0.
+2. You should have `wsl` installed, if not run `wsl --install`.
+3. Knowledge of fuelVM and other fuel tools.
+4. Basic concept of blockchain technology.
+5. Writing, compiling, and deploying to any monolith blockchain (EVM or non-EVM compatible blockchain).
```

Comment:
I would like to understand: why specifically _monolithic_ blockchain knowledge is required?

Reply:
Monolith as in a standalone blockchain I already explained in the braces (EVM or non-EVM). I've removed it for simplicity